ui/root/src/commonMain/kotlin/social/androiddev/root/splash/DefaultSplashComponent.kt

```diff
@@ -16,15 +16,4 @@ import com.arkivanov.decompose.ComponentContext
 
 class DefaultSplashComponent(
```

Comment:
I left a splash component just in case it'll be need in future, but I can drop it if required

Reply:
We can leave it for now in case we need to do kick off some task before taking the user to the first screen.